feat: DIA-1410: Add support for list of str for multilabel

robot-ci-heartex opened the pull request:
Hi @hakan458!

This PR was automaticaly generated via Follow Merge. 
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.

- https://github.com/HumanSignal/label-studio-sdk/pull/313 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/11003849778)

Commit:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11003849778

matt-bernstein committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11017329870

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/11017487930)
> Successfully merged:  create mode 100644 web/libs/editor/src/stores/Comment/Anchor.js

matt-bernstein committed:
Merge branch 'develop' into 'fb-dia-1410'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11017487930

Commit:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11021379452

hakan458 commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/11021611620)
> Successfully merged:  create mode 100644 docs/themes/v2/source/images/releases/2_16_video.png

hakan458 committed:
Merge branch 'develop' into 'fb-dia-1410'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11021611620

hakan458 committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/11021664125
